lib/resources/xinetd.rb

```diff
@@ -53,12 +53,14 @@ def read_content(path = @conf_path)
       return @contents[path] if @contents.key?(path)
       file = inspec.file(path)
       if !file.file?
-        return skip_resource "Can't find file \"#{path}\""
+        skip_resource "Can't find file \"#{path}\""
+        return nil
       end
 
       @contents[path] = file.content
       if @contents[path].nil? || @contents[path].empty?
-        return skip_resource "Can't read file \"#{path}\""
+        skip_resource "Can't read file \"#{path}\""
```

Comment:
Same as above - change to a raised exception, and then we can remove the `return nil`